packages/core/contracts/financial-templates/expiring-multiparty/Liquidatable.sol

```diff
@@ -504,6 +504,28 @@ contract Liquidatable is PricelessPositionManager {
         return liquidations[sponsor];
     }
 
+    /**
+     * @notice Accessor method to calculate a transformed collateral requirement using the finanical product library
+      specified during contract deployment. If no library was provided then no modification to the collateral requirement is done.
+     * @param price input price used as an input to transform the collateral requirement.
+     * @return transformedCollateralRequirement collateral requirement with transformation applied to it.
+     * @dev This method should never revert.
+     */
+    function transformCollateralRequirement(FixedPoint.Unsigned memory price)
+        public
+        view
+        returns (FixedPoint.Unsigned memory)
+    {
+        if (address(financialProductLibrary) == address(0)) return collateralRequirement;
+        try financialProductLibrary.transformCollateralRequirement(price, collateralRequirement) returns (
```

Comment:
Have you considered adding a test that tests what happens when `financialProductLibrary. transformCollateralRequirement` reverts?  I did something similar when testing what would happen when the perpetual's `withdrawFundingRateFees()` method reverts [here](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/test/MockPerpetual.sol#L48)

Reply:
I have not tested such but can add a mock and a test to do it. will report back once it's added.

Reply:
I've added tests for reverting library to liquidatable & retroactively to the priceless position manager to verify this try/catch logic works as expected.